doc/usecases/visitor_chat.md

```diff
@@ -0,0 +1,26 @@
+# Use Case: Visitor chat
+
+## Summary
+
+- **Scope:** Map View
+- **Level:** User Goal
+- **Actors:** 
+  - Owner: App User who owns the map
+  - Visitor: App User who visits the map
+- **Brief:** The user can interact with a visitor chat.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  The visitor has opened the app and selected a map.
+- **Main success scenario:**
+  - The visitor is automatically greeted by the chat. 
+  - The visitor can interact with the owner and other current visitors via the chat.
```

Comment:
Interactions should also include the ability to leave a like on the whole map, complementary to leaving likes on specific parts of the map as in the [photo layer](layers/photo_layer.md).

Reply:
Good suggestion but I think this would be a different use case